app/mailers/user_mailer.rb

```diff
@@ -0,0 +1,11 @@
+class UserMailer < ApplicationMailer
+  def account_activation(user)
+    @user = user
+    mail to: user.email, subject: "Account Activation"
+  end
+
+  def password_reset
+    @greeting = "Hi"
+    mail to: "to@example.org", subject: "Reset Password"
```

Comment:
the to address is incorrect.

Reply:
where is the password reset LINK?

Reply:
I was hoping to handle this when I do the password reset chapter